libfoundation/src/foundation-java-private.cpp

```diff
@@ -1460,6 +1465,8 @@ bool MCJavaCreateInterfaceProxy(MCNameRef p_class_name, MCTypeInfoRef p_signatur
                                                     t_interface,
                                                     t_handler);
     
+    t_proxy = s_env->PopLocalFrame(t_proxy);
+    
     MCJavaObjectRef t_result_value;
     if (!MCJavaObjectCreateNullable(t_proxy, t_result_value))
```

Comment:
Hmm... if you pop the frame after this you have a global ref so don't end up leaving a local ref lingering in the previous frame.

Reply:
Good point! Not sure why I did that :-)